extensions/blocks/testimonial/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import TestimonialEdit from './edit';
+import TestimonialSave from './save';
+import { __, _x } from '../../utils/i18n';
+
+export const name = 'testimonial';
+
+export const settings = {
+	title: __( 'Testimonial' ),
+	description: __( 'Lets you display testimonials from customers and clients.' ),
+	icon: 'embed-photo',
+	category: 'jetpack',
+	keywords: [ __( 'testimonial' ) ],
```

Comment:
You shouldn't need to duplicate this if it's the name of the Block itself? Some useful alternative keywords could be

https://www.merriam-webster.com/thesaurus/testimonial